cli/command_repository_connect.go

```diff
@@ -72,6 +73,7 @@ func (c *connectOptions) setup(svc appServices, cmd *kingpin.CmdClause) {
 	cmd.Flag("override-username", "Override username used by this repository connection").Hidden().StringVar(&c.connectUsername)
 	cmd.Flag("check-for-updates", "Periodically check for Kopia updates on GitHub").Default("true").Envar(svc.EnvName(checkForUpdatesEnvar)).BoolVar(&c.connectCheckForUpdates)
 	cmd.Flag("readonly", "Make repository read-only to avoid accidental changes").BoolVar(&c.connectReadonly)
+	cmd.Flag("permissive-index-reads", "Do not fail reading bad index entries").BoolVar(&c.connectPermissiveIndexReads)
```

Comment:
Also, might be good to signal here that this works only with the read-only flag.

Reply:
Also would be good to keep this flag hidden for now,